examples/drivenets/srlinux.cfg

```diff
@@ -0,0 +1 @@
+../multivendor/srlinux.cfg
```

Comment:
prefer just copying the config here in case paths change in future

Reply:
ok, i copied the config